Source/WebCore/dom/FullscreenOptions.idl

```diff
@@ -32,7 +32,7 @@
 };
 
 [
-    Conditional=FULLSCREEN_API, EnabledBySetting=FullScreenKeyboardLock
+    Conditional=FULLSCREEN_API
```

Comment:
Why are we removing the `EnabledBySetting` here? I think we would want this to be controllable by the User Agent in Page Settings, so we could allow keyboard lock on gaming sites, but prevent general sites without this need from capturing the keyboard.

I propose we leave this in place for this patch, and discuss moving the setting to `stable` in a follow-up patch. It's already marked as `testable` so your test cases should be fine even without this bit of your change.

Reply:
@brentfulgham  Agreed on your review. Previously, I just did not want to the default test case ended up timed out. 
But that's only the concern I had and I thought this might be better to leave to be timed out in case of default test.